service-runner/src/main/resources/dev-tools/docker-compose-kafka-sasl.yaml

```diff
@@ -0,0 +1,46 @@
+version: '3'
+services:
+  zookeeper:
```

Comment:
Do we need Zookeeper to test the new auth methods?

Reply:
This is dev tools and not used in the pipeline. Kraft configuration is tricky and was not working well with SASL_SSL auth. May need to spend more time to replace zookeeper with `kraft`.